impl.go

```diff
@@ -34,9 +62,9 @@ var (
 // If an unqualified interface such as "UserDefinedInterface" is given, then
 // the interface definition is presumed to be in the package within srcDir and
 // findInterface returns "", "UserDefinedInterface".
-func findInterface(iface string, srcDir string) (path string, id string, err error) {
-	if len(strings.Fields(iface)) != 1 {
-		return "", "", fmt.Errorf("couldn't parse interface: %s", iface)
+func findInterface(iface string, srcDir string) (path string, id string, typeParams []string, err error) {
```

Comment:
docs about typeParams would be good. What params does `[T, U any]` yield? What about `[T any, U any]`?